source/MaterialXView/Viewer.cpp

```diff
@@ -2204,6 +2204,7 @@ void Viewer::draw_contents()
     }
 
     // Render the current frame.
+    _renderPipeline->_frame = (_renderPipeline->_frame + 1) % 32768;
```

Comment:
Same as above, where `std::numeric_limits<unsigned int>::max() - 1` may be a better match than `32768`.

Reply:
Using modulo with the counter type's maximum.